config/management/genesis/src/verify.rs

```diff
@@ -239,15 +238,17 @@ fn validator_config(
     validator_account: AccountAddress,
     reader: Arc<dyn DbReader>,
 ) -> Result<ValidatorConfig, Error> {
-    let blob = reader
-        .get_latest_state_value(StateKey::AccountAddressKey(
-            account_config::validator_set_address(),
-        ))
-        .map_err(|e| Error::UnexpectedError(format!("ValidatorSet Account issue {}", e)))?
-        .ok_or_else(|| Error::UnexpectedError("ValidatorSet Account not found".into()))?;
-    let account_state = AccountState::try_from(&blob)
-        .map_err(|e| Error::UnexpectedError(format!("Failed to parse blob: {}", e)))?;
-    let validator_set: ValidatorSet = account_state
+    let account_state_view = AccountStateView {
```

Comment:
This can be
```
let account_view = reader
    .lastest_state_view()
    .as_account_view(validator_set_address());
```
if you do it as my other comment suggests.